### PR TITLE
Feat/dynamic model

### PR DESCRIPTION
**Pull request by mamoonraja**
The [core](https://github.com/IBM/unity-sdk-core/pull/18) was updated to add dynamic models, this pr regenerate SDK and uses dynamic models class for additional properties.

### CODE REVIEW

File: Examples/GenericSerialization.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ void Start()
             //Log.Debug("GenericSerialization", "main skill: {0}", uDefinedObject["name"]);
             //Log.Debug("GenericSerialization", "test: {0}", messageResponse);
 
-            var name = messageResponse.Context.Skills["main skill"]["user_defined"]["name"].ToString();
+            var name = messageResponse.Context.Skills.Skill.UserDefined["name"].ToString();
```
Comment (mediumTaj) [inline Examples/GenericSerialization.cs]: does this work here?
```cs
messageResponse.Context.Skills.Skill.UserDefined.Get("name");
```

Comment (mamoonraja) [reply Examples/GenericSerialization.cs]: Yep it worked, but I had to make this change: https://github.com/watson-developer-cloud/unity-sdk/pull/594/files#diff-e8323f26d84418a6500938a8e5996cc1, for it to work

File: Examples/GenericSerialization.cs [hunk 2]
```diff
@@ -47,7 +47,7 @@ void Start()
             //Log.Debug("GenericSerialization", "main skill: {0}", uDefinedObject["name"]);
             //Log.Debug("GenericSerialization", "test: {0}", messageResponse);
```
Comment (mediumTaj) [inline Examples/GenericSerialization.cs]: ooh man, i think we can delete all of this commented code. sorry, lol!

Comment (mamoonraja) [reply Examples/GenericSerialization.cs]: deleted!

**Pull request merged**